WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -299,6 +300,36 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
   });
 }
 
+/**
+ * @return Current user interface style as a string
+ */
++ (NSString *)userInterfaceStyle
+{
+  static id userInterfaceStyle = nil;
+  static dispatch_once_t styleOnceToken;
+  dispatch_once(&styleOnceToken, ^{
+    if ([UITraitCollection respondsToSelector:NSSelectorFromString(@"currentTraitCollection")]) {
+      id currentTraitCollection = [UITraitCollection performSelector:NSSelectorFromString(@"currentTraitCollection")];
+      if (nil != currentTraitCollection) {
+        userInterfaceStyle = [currentTraitCollection valueForKey:@"userInterfaceStyle"];
```

Comment:
I noticed `[[UITraitCollection currentTraitCollection] userInterfaceStyle]` does not cahnge dynamically...

It returned `1` on an iOS 13.3 simulator when dark mode off. Then, I enabled the dark mode from developer option without stopping WDA. Then, the `userInterfaceStyle` returned `1`. But when I restart the WDA once, the return value became `2`.

This behaviour is not by WDA, but let me leave it here.

Reply:
unfortunately I was not able to find anything there, which could allow us to get this value without having callbacks on traits change (which I suppose would not be possible to implement inside WDA lib). @KazuCocoa Do you have any other ideas on which API could be used for this purpose?

Reply:
No, unfortunately... I also found only this method to get the current UI style